addon/templates/components/themed-syntax.hbs

```diff
@@ -2,4 +2,4 @@
   NOTE: Don't update this layout/spacing/tabbing
   This format allows for the best looking whitespace in codeblocks
 }}
-<code class="code hljs {{if isDark 'dark'}}">{{#unless code}}{{yield}}{{else}}  {{{code}}}{{/unless}}</code>
\ No newline at end of file
+<code class="code hljs {{if isDark 'dark'}}">{{#unless code}}{{yield}}{{else}}  {{{highlightedCode}}}{{/unless}}</code>
```

Comment:
@pfmmfp I think we can probably use `{{html-safe}}` instead of triple braces these days.

Reply:
Done!